common3.json

```diff
@@ -8587,207 +8587,207 @@
     },
     {
       "id" : 62158,
-      "text" : "サポーター２"
+      "text" : "Supporter 2"
     },
     {
       "id" : 62159,
-      "text" : "シナリオ１"
+      "text" : "Scenario 1"
     },
     {
       "id" : 62160,
-      "text" : "シナリオ２"
+      "text" : "Scenario 2"
     },
     {
       "id" : 62161,
-      "text" : "シナリオ３"
+      "text" : "Scenario 3"
     },
     {
       "id" : 62162,
-      "text" : "シナリオ４"
+      "text" : "Scenario 4"
     },
     {
       "id" : 62163,
-      "text" : "シナリオ５"
+      "text" : "Scenario 5"
     },
     {
       "id" : 62164,
-      "text" : "シナリオ６"
+      "text" : "Scenario 6"
     },
     {
       "id" : 62165,
-      "text" : "シナリオ７"
+      "text" : "Scenario ７"
     },
     {
       "id" : 62166,
-      "text" : "シナリオ８"
+      "text" : "Scenario ８"
     },
     {
       "id" : 62167,
-      "text" : "シナリオ９"
+      "text" : "Scenario ９"
```

Comment:
there's a lot like these so I suggest to double check, but you do not want to keep the original "９", you just want to write a normal "9" because although they look similar, they're known to cause problems for some reasons so you really want to replace these numbers.

Reply:
Alright

Reply:
btw "Story Mode" is called "Scenario Mode" in Japanese so anything with scenario might be more accuractely translated as "Story" or "Story Mode"

Reply:
Should we do a search and replace while I fixup the jp characters?